roles/mongodb/tasks/install.yml

```diff
@@ -18,86 +10,100 @@
-#    state: directory
-#  when: internet_available and is_raspbian
-#
-#- name: Download & unzip MongoDB 3.0.14's 3 core binaries to /opt/iiab/downloads/mongodb-3.0.1x (raspbian)
-#  unarchive:
-#    remote_src: yes
-#    src: "{{ iiab_download_url }}/mongodb_stretch_3_0_14_core.zip"
-#    dest: "{{ downloads_dir }}/mongodb-3.0.1x"
-#  when: internet_available and is_raspbian
-#
-#- name: Install (copy) 3 binaries from /opt/iiab/downloads/mongodb-3.0.1x/core to /usr/bin (raspbian)
-#  copy:
-#    src: "{{ item }}"
-#    dest: /usr/bin
-#  with_fileglob:
-#    - "{{ downloads_dir }}/mongodb-3.0.1x/core/*"
-#  when: internet_available and is_raspbian
-#
-#- name: Download & unzip MongoDB 3.0.15's 9 tools binaries to /opt/iiab/downloads/mongodb-3.0.1x (raspbian)
-#  unarchive:
-#    remote_src: yes
-#    src: "{{ iiab_download_url }}/mongodb_stretch_3_0_14_tools.zip"
-#    dest: "{{ downloads_dir }}/mongodb-3.0.1x"
-#  when: internet_available and is_raspbian
-#
-#- name: Install (copy) 9 binaries from /opt/iiab/downloads/mongodb-3.0.1x/tools to /usr/bin (raspbian)
-#  copy:
-#    src: "{{ item }}"
-#    dest: /usr/bin
-#  with_fileglob:
-#    - "{{ downloads_dir }}/mongodb-3.0.1x/tools/*"
-#  when: internet_available and is_raspbian
-
-- name: Create Linux group mongodb (raspbian)
-  group:
-    name: mongodb
-    state: present
-  when: is_raspbian | bool
-
-- name: Create Linux user mongodb (raspbian)
-  user:
-    name: mongodb
-    group: mongodb    # primary group
-    groups: mongodb
-    home: /var/lib/mongodb
-    shell: /usr/sbin/nologin
-  when: is_raspbian | bool
+- block:
+  - name: Create dir /tmp/mongodb-3.0.1x (raspbian)
+    file:
+      path: /tmp/mongodb-3.0.1x
+      state: directory
+
+  - name: Download & unzip 20MB http://download.iiab.io/packages/mongodb_stretch_3_0_14_core.zip to /tmp/mongodb-3.0.1x (raspbian)
+    unarchive:
+      remote_src: yes
+      src: "{{ iiab_download_url }}/mongodb_stretch_3_0_14_core.zip"
+      dest: /tmp/mongodb-3.0.1x
+
+  - name: Install (move) its 3 CORE binaries from /tmp/mongodb-3.0.1x/core to /usr/bin (raspbian)
+    shell: mv /tmp/mongodb-3.0.1x/core/* /usr/bin
+
+  - name: Download & unzip 15MB http://download.iiab.io/packages/mongodb_stretch_3_0_14_tools.zip [IN FACT THIS ONE'S 3.0.15] to /tmp/mongodb-3.0.1x (raspbian)
+    unarchive:
+      remote_src: yes
+      src: "{{ iiab_download_url }}/mongodb_stretch_3_0_14_tools.zip"
+      dest: /tmp/mongodb-3.0.1x
+
+  - name: Install (move) its 9 TOOLS binaries from /opt/iiab/downloads/mongodb-3.0.1x/tools to /usr/bin (raspbian)
+    shell: mv /tmp/mongodb-3.0.1x/tools/* /usr/bin
+
+  - name: Create Linux group mongodb (raspbian)
+    group:
+      name: mongodb
+      state: present
+
+  - name: Create Linux user mongodb (raspbian)
+    user:
+      name: mongodb
+      group: mongodb    # primary group
+      groups: mongodb
+      home: /var/lib/mongodb
+      shell: /usr/sbin/nologin
+
+  - name: Install /etc/mongod.conf from template
```

Comment:
Is installing `/etc/mongod.conf` from template not needed on 64-bit OS's?

As Line 62's end-of-block seems to imply?

(If so, any idea why?)

Reply:
The format of the file changed, the template breaks mongodb version 4.4